ml-agents/mlagents/trainers/cli_utils.py

```diff
@@ -168,6 +168,13 @@ def _create_parser() -> argparse.ArgumentParser:
         action=DetectDefaultStoreTrue,
         help="Forces training using CPU only",
     )
+    argparser.add_argument(
+        "--torch",
+        default=False,
+        action=DetectDefaultStoreTrue,
+        help="(Experimental) Use the PyTorch framework instead of TensorFlow. Install PyTorch "
```

Comment:
Is there a way to hide the help, or do we not care if the users see this?

Reply:
Yes - `argparse.SUPPRESS`. I can hide it. Should we?

Reply:
Let's not hide it then